backend/api/routes/notifications.js

```diff
@@ -11,6 +11,7 @@ router.route("/").get(authenticateMiddleware, async (req, res) => {
       .from("notifications")
       .select("*")
       .eq("user_id", user_id)
+      .neq("status", "read")
```

Comment:
Should we instead order them by this status to display the read ones at the bottom?